internal/scout/usage/k8s.go

```diff
@@ -362,7 +453,7 @@ func getStorageUsage(
 
 	exec, err := remotecommand.NewSPDYExecutor(cfg.restConfig, "POST", req.URL())
 	if err != nil {
-		return "", err
+		return -1, -1, err
```

Comment:
I think these should return the zero value for `float64` by convention which would be `0`